subsys/ipc/ipc_service/backends/Kconfig.icmsg

```diff
@@ -0,0 +1,16 @@
+# Copyright (c) 2022 Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
+
+if IPC_SERVICE_BACKEND_ICMSG
+
+config IPC_SERVICE_BACKEND_ICMSG_CB_BUF_SIZE
+	int "Size of static callback buffer size"
+	range 1 255
```

Comment:
Implementation limitation. Memory abstraction is written in a way that stores messages. Message is encapsulated with prefixing message len that is one byte. I didn't want to reserve two bytes for that - simply because the use case we have is mostly short messages ~10-30 Bytes. fyi rpmsg has 512 Bytes limitation.

Reply:
We have considered implementing expandable size type but decided it's overkill now.

As for reserving the value, I don't think it is needed as long as code will recompile for both cores. I understand you would like to play it safe in case somebody would like to update FW one one core only. The thing is old FW would not understand the reserved value anyway so new FW would not be able to use it. It will also not use the reserved value itself. Essentially we would need update on both cores anyway.

I don't think we need to reserve the value from protocol point of view.

Reply:
> Essentially we would need update on both cores anyway.

Fair enough

Reply:
I saw it brought many doubts. I expanded message length to `uint16_t` so is now in `range 1 65535`